packages/core/admin/utils/get-plugins-path.js

```diff
@@ -1,15 +1,23 @@
 'use strict';
 
-const { join, resolve } = require('path');
+const { join, resolve, sep, posix } = require('path');
 const fs = require('fs-extra');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const glob = require('glob');
 
 // Only for dev environement
 const getPluginsPath = () => {
   const rootPath = resolve(__dirname, '..', join('..', '..', '..', 'packages'));
-  const corePath = join(rootPath, 'core', '*');
-  const pluginsPath = join(rootPath, 'plugins', '*');
+  /**
+   * So `glob` only supports '/' as a path separator, so we need to replace
+   * the path separator for the current OS with '/'. e.g. on windows it's `\`.
+   *
+   * see https://github.com/isaacs/node-glob/#windows for more information
+   *
+   * and see https://github.com/isaacs/node-glob/issues/467#issuecomment-1114240501 for the recommended fix.
+   */
+  const corePath = join(rootPath, 'core', '*').split(sep).join(posix.sep);
+  const pluginsPath = join(rootPath, 'plugins', '*').split(sep).join(posix.sep);
```

Comment:
This now breaks on posix paths that contain a backslash `\`

I'm not sure if we can handle this without checking the platform and handling it differently on Windows. Otherwise we're going to run into problems with __dirname being absolute and trying to come up with our own hacks. But there may be some way to coerce everything properly using path.posix.normalize and path.win32.normalize without specifically checking platform.

Reply:
I tested the fix on Windows OS and the build worked just fine, and so does the yarn setup

Reply:
I think it's probably simpler to add some tests. So a path could look like this – `User/Josh/code/strapi/packages/core/*` or `C:\\User\\Josh\\code\\strapi\\packages\\core\\*`

Are there any others @innerdvations?

Reply:
A path could also look like this: `User/Josh/code/strapi\\stuff/packages/core/*` where `strapo\\stuff` is the name of a single directory. That's the main issue with doing a replacement without checking the platform.